CHANGELOG.md

```diff
@@ -69,6 +69,12 @@
 
 ## v1.33.0 - TBD
 
+## v1.32.1 - 2021-10
+
+### [Pub/Sub](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/pubsub/README.md)
+
+* fix(pubsub): do not overwrite ConnectionOptions ([#7406](https://github.com/googleapis/google-cloud-cpp/pull/7406))
```

Comment:
I should have called this earlier: this is what it does not do, but there are many things it does not do.  Maybe "respect non-default values in pubsub::ConnectionOptions"?